app/jest/layout.test.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import Layout from 'layout/Layout';
+
+const handleLogin = jest.fn();
+const handleLogout = jest.fn();
+
+function LayoutComponent() {
+  return (
+    <Layout session={session} user={null} enableGold={true} onLoginClick={handleLogin} onLogoutClick={handleLogout} />
+  );
+}
+
+const ROCKET_CHAT_HYPERLINK = 'https://chat.developer.gov.bc.ca/channel/sso';
+const PATHFINDER_SSO_HYPERLINK = 'mailto:bcgov.sso@gov.bc.ca';
+const DOCUMENTATION_HYPERLINK = 'https://github.com/bcgov/sso-keycloak/wiki';
+const DISCLAIMER_HYPERLINK = 'https://www2.gov.bc.ca/gov/content/home/disclaimer';
+const PRIVACY_HYPERLINK = 'https://www2.gov.bc.ca/gov/content/home/privacy';
+const ACCESSIBILITY_HYPERLINK = 'https://www2.gov.bc.ca/gov/content/home/accessible-government';
+const COPYRIGHT_HYPERLINK = 'https://www2.gov.bc.ca/gov/content/home/copyright';
+
+const session = {
```

Comment:
@kurokn please ignore redundant code, you can make use of session object from `home.test.tsx`